modules/ROOT/pages/apicat-create-descriptor-file-cli.adoc

```diff
@@ -67,6 +67,36 @@ The presence and content of the designated descriptor file affects the result of
 
 * Existing file with content: The command fails and shows an error message that says the descriptor file already exists.
 
+=== Add Images through a Descriptor File
+
+You can add images to your APIs by adding a markdown file in the descriptor with a link to an image (.png, .jpg). You can visualize the image in the documentation page created for the Exchange Asset.
+
+Example output that is written to the descriptor file:
+
+----
+#%Catalog Descriptor 1.0
+projects:
+  - main: sources/api.raml
+    documentation:
+      - Summary: Summary.md
+      - Use Cases: Use_Cases.md
+    assetId: version-basic-api
+    apiVersion: v4
+    version: 4.0.0
+----
+With the `summary.md` file containing an image and a documentation markdown file:
+----
+This is a caching service that can be used to retrieve picklist values from publishers who wish to provide a caching service for their picklist fields.
+
+![muley logo](muleyLogo.png)
+
+[Read all the Use Cases](Use_Cases.md)
+----
+
+The presence and content of the designated descriptor file affect the result of the command as follows:
+
+* No documentation entry in the descriptor catalog pointing to the readme file: the link to move between pages in the Asset's portal doesn't work.
```

Comment:
Actually, with this input, the image will be embedded in the Summary documentation page in Exchange, and it will also have a link to the Use Cases documentation page.

The link does not work only if the `Use Cases` entry is not present in the catalog descriptor `documentation` entry. In order for the link to work, there must be at least one entry in the `documentation` entry in the catalog descriptor with the referenced markdown file